java_8/src/test/java/android/BrowserStackSample.java

```diff
@@ -2,21 +2,21 @@
 
 import io.appium.java_client.AppiumBy;
 import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.options.UiAutomator2Options;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.time.Duration;
 import java.util.HashMap;
 import java.util.List;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 public class BrowserStackSample {
 
   public static void main(String[] args)
-    throws MalformedURLException, InterruptedException {
-    DesiredCapabilities caps = new DesiredCapabilities();
+      throws MalformedURLException, InterruptedException {
+    UiAutomator2Options options = new UiAutomator2Options();
```

Comment:
Please use Mutuable Caps?

Reply:
This has been discussed with product team and @francisf and we decided to use Options.

Mutuable caps are not needed here, for android and ios separate folders are present because a single script cannot be used for both ios and android automation.